tests/src/test/java/tachyon/client/FileInStreamIntegrationTest.java

```diff
@@ -69,6 +70,21 @@ public static final void beforeClass() throws Exception {
     sWriteBoth = StreamOptionUtils.getCreateFileOptionsCacheThrough(sTachyonConf);
     sWriteTachyon = StreamOptionUtils.getCreateFileOptionsMustCache(sTachyonConf);
     sWriteUnderStore = StreamOptionUtils.getCreateFileOptionsThrough(sTachyonConf);
+    sTestPath = PathUtils.uniqPath();
+    for (int k = MIN_LEN; k <= MAX_LEN; k += DELTA) {
```

Comment:
Instead of a utils method it would be better to have some kind of inheritance for the stream tests, as they share a lot of common logic, but I think its a separate improvement from this solution.